_posts/2023-02-16-secure-cloud-foundation.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+authors: [lander_marien, duncan_casteleyn]
+title: 'Secure cloud foundation tooling'
+image: /img/2023-02-16-secure-cloud-foundation/header.png # TODO - change image
+tags: [aws, cloud]
+category: Cloud
+comments: true
+---
+
+<!-- TODO complete index -->
+
+- [Introduction](#introduction)
+- [The Warden: Open Policy Agent](#the-warden-open-policy-agent)
+- [The Janitor: Cloud Custodian](#the-janitor-cloud-custodian)
+- [The Watchdog: Snyk](#the-watchdog-snyk)
+- [The Container Detective: Trivy](#the-container-detective-trivy)
+- [The Composer: Fugue](#the-composer-fugue)
+
+# Introduction
+
+If you have ever come in to contact with an enterprise cloud environment,
+you know that keeping it secure and compliant can be a cumbersome task.
+
+Many companies have their own security and compliance requirements,
+and they should at least follow a best practice standard like CIS AWS Foundations Benchmark.
+
+You'll find out that because access to the cloud is splintered over so many teams and projects,
```

Comment:
```suggestion
Often, you'll find that because of the segmented responsibilities across many teams and projects, gaps exist in the security and compliancy posture of the organisation. 
```